src/main/prometheus/prometheus-provider-registry.injectable.ts

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
1. Same naming as before, no changes here.
2. Preference over unique file names leads to better dev-experience because of discoverability.

Does your IDE have auto import? :)